grpc/src/tx.rs

```diff
@@ -85,15 +87,11 @@ where
     <T::ResponseBody as Body>::Error: Into<StdError> + Send,
     S: DocSigner,
 {
-    /// Create a new transaction client.
-    pub async fn new(
-        transport: T,
-        account_address: &Address,
-        account_pubkey: VerifyingKey,
-        signer: S,
-    ) -> Result<Self> {
+    /// Create a new transaction client with the specified account.
+    pub async fn new(transport: T, account_pubkey: VerifyingKey, signer: S) -> Result<Self> {
         let client = GrpcClient::new(transport);
-        let account = client.get_account(account_address).await?;
+        let account_address = Address::from_account_veryfing_key(account_pubkey);
```

Comment:
sorry for the late notice, but I think we should just use `AccAddress` everywhere directly